source/_includes/code_examples/code_example.liquid

```diff
@@ -3,25 +3,25 @@
   style="--split-location: {{ code.splitLocation }}%"{% endif %}
 >
   <ul class="ui-tabs-nav ui-helper-reset ui-helper-clearfix">
-    {% if code.scss.size %}
-      {% render 'code_examples/tab', name: 'SCSS', syntax: 'scss', id: exampleName, enabled: true %}
-    {% endif %}
-    {% if code.sass.size %}
-      {% assign enabled = code.scss.size == 0 %}
-      {% render 'code_examples/tab', name: 'Sass', syntax: 'sass', id: exampleName, enabled: enabled %}
-    {% endif %}
-    {% if code.css.size %}
-      {% render 'code_examples/tab', name: 'CSS', syntax: 'css', id: exampleName, enabled: false %}
-    {% endif %}
+    {%- if code.scss.size -%}
+      {%- render 'code_examples/tab', name: 'SCSS', syntax: 'scss', id: exampleName, enabled: true -%}
+    {%- endif -%}
+    {%- if code.sass.size -%}
+      {%- assign enabled = code.scss.size == 0 -%}
+      {%- render 'code_examples/tab', name: 'Sass', syntax: 'sass', id: exampleName, enabled: enabled -%}
+    {%- endif -%}
+    {%- if code.css.size -%}
+      {%- render 'code_examples/tab', name: 'CSS', syntax: 'css', id: exampleName, enabled: false -%}
+    {%- endif -%}
   </ul>
-  {% if code.scss.size %}
-    {% render 'code_examples/panel', name: 'SCSS Syntax', syntax: 'scss', id: exampleName, examples: code.scss, paddings: code.scssPaddings, enabled: true %}
-  {% endif %}
-  {% if code.sass.size %}
-    {% assign enabled = code.scss.size == 0 %}
-    {% render 'code_examples/panel', name: 'Sass Syntax', syntax: 'sass', id: exampleName, examples: code.sass, paddings: code.sassPaddings, enabled: enabled %}
-  {% endif %}
-  {% if code.css.size %}
-    {% render 'code_examples/panel', name: 'CSS Output', syntax: 'css', id: exampleName, examples: code.css, paddings: code.cssPaddings, enabled: false %}
-  {% endif %}
+  {%- if code.scss.size -%}
+    {%- render 'code_examples/panel', name: 'SCSS Syntax', syntax: 'scss', id: exampleName, examples: code.scss, paddings: code.scssPaddings, enabled: true -%}
+  {%- endif -%}
+  {%- if code.sass.size -%}
+    {%- assign enabled = code.scss.size == 0 -%}
+    {%- render 'code_examples/panel', name: 'Sass Syntax', syntax: 'sass', id: exampleName, examples: code.sass, paddings: code.sassPaddings, enabled: enabled -%}
+  {%- endif -%}
+  {%- if code.css.size -%}
+    {%- render 'code_examples/panel', name: 'CSS Output', syntax: 'css', id: exampleName, examples: code.css, paddings: code.cssPaddings, enabled: false -%}
+  {%- endif -%}
```

Comment:
These changes allow us to use `{% codeExample %}` in md files... mostly. There's still a rogue `<p></p>` tag in between panels. Notice how a blank line appears when you switch to the Sass tab in [this example](https://deploy-preview-51--sass-site-oddbird.netlify.app/documentation/breaking-changes/strict-unary/). Not sure how to fix it 😕

Reply:
Hrm, yeah, shucks -- that makes me a bit nervous. The code examples intentionally add newlines to create padding, but I think markdown is turning those newlines into extra `<p>` tags. There are seems to be another issue with indentation:

![Screenshot 2023-05-25 at 10 48 57 PM](https://github.com/oddbird/sass-site/assets/552316/f63f8b75-663f-402a-9138-8a0a2fc38eba)